inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/SortClusterService.java

```diff
@@ -32,4 +32,6 @@ public interface SortClusterService {
      * @return Corresponding response.
      */
     SortClusterResponse getClusterConfig(String clusterName, String md5);
+
```

Comment:
It's better to add docs for new added interface.

Reply:
Done.